hashicorp/boundary-ui
# Icu 12027 let the user know if the dc isnt compatible with the controller and link them to the previous version

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12027)

## Description



## Screenshots (if appropriate):
<img width="675" alt="Screenshot 2024-01-08 at 2 02 08 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/13661ab6-3643-4d1c-8f78-a6fc4834f0a0">


If the download fails we will show the user an error alert and direct them to the releases page for 1.7.1:
<img width="675" alt="Screenshot 2024-01-08 at 2 02 46 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/f34b85db-53cb-4dd8-a506-79e1c9670d7d">



## How to Test


use command: ENABLE_MIRAGE=false yarn start:desktop

llb-list-pagination-2 now has pagination for scopes so if using the most recent changes from that branch will not show this new modal. So use any public version of the binary will allow for the new modal to appear.

## Checklist:
